feat: Add delete asset function

cliff-comvex opened the pull request:
## Description:
Add ability to delete UiPath asset by ID.

## Added
- DeleteByID function in asset handler

## References:
https://comvex.myjetbrains.com/youtrack/issue/DGM2-8728

Commit:
feat: Add delete asset function

Stefancomvex reviewed (approved):
Looks good to me. Looks like you also added the ability to specify the folder to all the other functions?

cliff-comvex commented:
> Looks good to me. Looks like you also added the ability to specify the folder to all the other functions?

@Stefancomvex it has been there, I just extracted it to a variable to easier to update.
